dht_test.go

```diff
@@ -1401,14 +1405,17 @@ func testFindPeerQuery(t *testing.T,
 	for i := 0; i < bootstrappers; i++ {
 		for j := 0; j < bootstrapperLeafConns; j++ {
 			v := mrand.Intn(leafs)
-			connect(t, ctx, others[i], others[bootstrappers+v])
+			connectNoSync(t, ctx, others[i], others[bootstrappers+v])
 		}
 	}
 
 	for i := 0; i < bootstrappers; i++ {
-		connect(t, ctx, guy, others[i])
+		connectNoSync(t, ctx, guy, others[i])
 	}
 
+	// give some time for things to settle down
+	waitForWellFormedTables(t, dhts, minRTRefreshThreshold, minRTRefreshThreshold, 5*time.Second)
```

Comment:
this seems ok for now, but it's effectively equivalent to `time.Sleep(5 * time.Second)` right? There's no reason to expect that these reach `minRTRefreshThreshold` peers since they're not in the routing table. What if we did `min(bootstrapperLeafConns, minRTRefreshThreshold)` or something like that?

Reply:
But, repeated `fixLowPeers` should get us to `minRTRefreshThreshold `, right ?

Reply:
Ya, I'm being silly. All these tests should have `minRTRefreshThreshold` peers in the tests. I guess the "correct" version of this is more like `min(minRTRefreshThreshold, len(dhts)-1)`, but if the DHT is small enough this will fail too...